site/src/docs/patterns/cookies/basics.mdx

```diff
@@ -0,0 +1,114 @@
+| No cookies at all                   | No                             | No actions needed.                                                                                                  
+[Table 1:Cases when the consent is needed and when it is not]|
+
+Refer to the next chapter to learn how to categorise your cookies and decide which cookies should be necessary and which should not.
+
+## Cookie information and categorisation
+
+The cookie consent needs to be asked when your service becomes public (even during alpha/beta periods). [Service under Hel.fi? / Service on its own?]
+
+To prepare to use the HDS cookie compliance component, you have to:
+1. List all the cookies related to your service
+2. Decide which cookies are necessary
+3. Categorise cookies into predefined categories
+
+We will go over each step in the following chapters.
+
+### 1. List all the cookies related to your service
+
+First, you need to list all the cookies your service is using. It is also a good moment to review the list of cookies. Aim to use as few cookies as possible and only gather the information your service **currently** needs. You should easily be able to justify the need for each cookie to the user.
+
+For each cookie, you need to gather and display the following information. This information will be displayed in the cookie consent window if the user wishes to get more information about a specific cookie.
+
+| Cookie data            | Description                                                                                       | Example |
+| ---------------------- | ------------------------------------------------------------------------------------------------- | ------- |
+| **Name**               | What is the name of the cookie file?                                                              | "cookie-agreed"
+| **Cookie set by**      | What domain is the provider of this cookie?                                                       | "hel.fi"
+| **Purpose of use**     | Why the cookie is needed? What purpose does it serve?                                             | "Used to store information about whether visitors have given or declined the use of cookie categories used on this service."
+| **Period of validity** | How long the cookie will be valid? This means time to expire from the creation of the cookie.     | "100 days"
+[Table 2:Information that needs to be shown for each cookie]|
+
+#### What if the user has given consent to my cookies in some other Helsinki service?
+
+Using common Helsinki cookie consent brings the benefit of sharing consents. If user gives consents at site A, same consents does not have to be asked again at site B. Sites sharing cookies must have the same domain. e.g. *.hel.fi. To learn more about common Helsinki service cookies, please refer to the <Link href="/patterns/cookies/common-helsinki-cookies">Common Helsinki cookies page</Link>.
```

Comment:
The link is broken in demo environment. Use this syntax instead to get the path prefix added:

`[Common Helsinki cookies page](/patterns/cookies/common-helsinki-cookies)`

Reply:
Fixed